mitype/app.py

```diff
@@ -121,6 +121,9 @@ def typing_mode(self, win, key):
 
         self.key_strokes.append([time.time(), key])
 
+        if key:
```

Comment:
Remove this if statement. Let the function be called even if no key is pressed, that way the stat is updated every frame.